lib/build.gradle

```diff
@@ -64,7 +64,7 @@ dependencies {
   implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:$kotlinCoroutinesVersion"
 
   // Awala
-  api 'tech.relaycorp:relaynet:[1.48.0,2.0.0)'
+  implementation 'tech.relaycorp:relaynet:[1.48.0,2.0.0)'
```

Comment:
With this change, I think we should start using a fixed version to ensure reproducibility going forward.